opslevel/datasource_opslevel_repository.go

```diff
@@ -23,20 +24,49 @@ type RepositoryDataSource struct {
 	CommonDataSourceClient
 }
 
+// LanguagesModel describes the model for the Languages data of the repository.
+type LanguagesModel struct {
+	Name  types.String `tfsdk:"name"`
+	Usage types.String `tfsdk:"usage"`
+}
+
 // RepositoryDataSourceModel describes the data source data model.
 type RepositoryDataSourceModel struct {
-	Alias types.String `tfsdk:"alias"`
-	Id    types.String `tfsdk:"id"`
-	Name  types.String `tfsdk:"name"`
-	Url   types.String `tfsdk:"url"`
+	Alias     types.String     `tfsdk:"alias"`
+	Id        types.String     `tfsdk:"id"`
+	Name      types.String     `tfsdk:"name"`
+	Url       types.String     `tfsdk:"url"`
+	Languages []LanguagesModel `tfsdk:"languages"`
+}
+
+// LanguagesValue function converts the raw opslevel data to terraform friendly format
+func LanguagesValue(value []opslevel.Language) []LanguagesModel {
+	var languages []LanguagesModel
+
+	if len(value) == 0 {
+		return []LanguagesModel{}
+	}
+
+	for _, lang := range value {
+		language := LanguagesModel{
+			Name:  types.StringValue(lang.Name),
+			Usage: types.StringValue(strconv.FormatFloat(float64(lang.Usage), 'f', -1, 32)),
```

Comment:
👋 @vini-mw 
This workaround helped to point out that `opslevel.Language.Usage` should really be a `float64` instead of a `float32`. We've [updated this in opslevel-go](https://github.com/OpsLevel/opslevel-go/pull/395) and also in this repo's oplevel-go submodule of the `main` branch. This allows `LanguagesModel.Usage` to be a `Float64` type now 🚀

Reply:
oh nice stuff!

i'll refactor the code to work with that right away then! 
thanks!

Reply:
aaand its done. thanks for handling this, i didnt like having that string conversion at all 😇